public/javascripts/SVValidate/src/keyboard/Keyboard.js

```diff
@@ -160,6 +206,98 @@ function Keyboard(menuUI) {
+                case 49: // "1" key
+                case 97: // Numpad "1" key
+                    if (menuUI.yesButton.hasClass('chosen')) {
+                        $('#severity-button-1').click();
+                        svv.tracker.push("KeyboardShortcut_Severity1", {
+                            keyCode: e.keyCode
+                        });
+                    }
+                    if (menuUI.yesButton.hasClass('chosen')) {
+                        $('#severity-button-1').click();
+                        svv.tracker.push("KeyboardShortcut_Severity1", {
+                            keyCode: e.keyCode
+                        });
+                    } else if (menuUI.noButton.hasClass('chosen')) {
+                        $('#no-button-1').click();
+                        svv.tracker.push("KeyboardShortcut_DisagreeReason1", {
+                            keyCode: e.keyCode
+                        });
+                    } else if (menuUI.unsureButton.hasClass('chosen')) {
+                        $('#unsure-button-1').click();
+                        svv.tracker.push("KeyboardShortcut_UnsureReason1", {
+                            keyCode: e.keyCode
+                        });
+                    }
+                    break;
+
+
+                case 50: // "2" key
+                case 98: // Numpad "2" key
+                    if (menuUI.yesButton.hasClass('chosen')) {
+                        $('#severity-button-2').click();
+                        svv.tracker.push("KeyboardShortcut_Severity2", {
+                            keyCode: e.keyCode
+                        });
+                    } else if (menuUI.noButton.hasClass('chosen')) {
+                        $('#no-button-2').click();
+                        svv.tracker.push("KeyboardShortcut_DisagreeReason2", {
+                            keyCode: e.keyCode
+                        });
+                    } else if (menuUI.unsureButton.hasClass('chosen')) {
+                        $('#unsure-button-2').click();
+                        svv.tracker.push("KeyboardShortcut_UnsureReason2", {
+                            keyCode: e.keyCode
+                        });
+                    }
+                    break;
+
+                case 51: // "3" key
+                case 99: // Numpad "3" key
+                    if (menuUI.yesButton.hasClass('chosen')) {
+                        $('#severity-button-3').click();
+                        svv.tracker.push("KeyboardShortcut_Severity3", {
+                            keyCode: e.keyCode
+                        });
+                    } else if (menuUI.noButton.hasClass('chosen')) {
+                        $('#no-button-3').click();
+                        svv.tracker.push("KeyboardShortcut_DisagreeReason3", {
+                            keyCode: e.keyCode
+                        });
+                    } else if (menuUI.unsureButton.hasClass('chosen')) {
```

Comment:
Something fun to trip you up here: Not all label types have three default options. CurbRamp labels, for example, have only two default options for the Unsure category, plus a text box. So the '3' key should go to the third option, which may be a default option, or it may be a text box.

In fact, we should make this more flexible. Let's try to not make assumptions about the number of available options, aside from having a maximum of 4 options (where one of them could be an text box). So I imagine that the same function could be called for all 4 number shortcuts, and within that function we decide what to do based off of how many buttons are available, etc etc.